orchestrator/services/subscriptions.py

```diff
@@ -29,6 +29,7 @@
     IP_PREFIX_SUBSCRIPTION_ID,
     NODE_SUBSCRIPTION_ID,
     PARENT_IP_PREFIX_SUBSCRIPTION_ID,
+    PEER_GROUP_SUBSCRIPTION_ID,
```

Comment:
We seem to leak some SURF specific constants. Maybe good to add least add a todo (in the code) to split/clean it?